BitwardenShared/Core/Tools/Repositories/ExportCXFCiphersRepository.swift

```diff
@@ -0,0 +1,129 @@
+import AuthenticationServices
+import BitwardenSdk
+
+/// Protocol for a repository to handle exporting ciphers in Credential Exchange Format.
+protocol ExportCXFCiphersRepository {
+    /// Builds the summary of the ciphers to export using `CXFCredentialsResult`.
+    /// - Parameter ciphers: Ciphers to build the summary from.
+    /// - Returns: An array of `CXFCredentialsResult` that has the summary of the ciphers to export by type.
+    func buildCiphersToExportSummary(from ciphers: [Cipher]) -> [CXFCredentialsResult]
```

Comment:
⛏ this make it much better to read. could you follow this styling and change the rest of the protocol definition as well?
```suggestion
    /// Builds the summary of the ciphers to export using `CXFCredentialsResult`.
    ///
    /// - Parameter ciphers: Ciphers to build the summary from.
    /// - Returns: An array of `CXFCredentialsResult` that has the summary of the ciphers to export by type.
    ///
    func buildCiphersToExportSummary(from ciphers: [Cipher]) -> [CXFCredentialsResult]
```